### PR TITLE
git updates

### PR DESCRIPTION
**Pull request by bakerboy448**
git

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+name: Bug Report
+title: "[BUG]: "
```
Comment (markus101) [inline .github/ISSUE_TEMPLATE/bug_report.yml:2]: The label provides this information after triaging.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+name: Feature Request
+title: "[FEAT]: "
```
Comment (markus101) [inline .github/ISSUE_TEMPLATE/feature_request.yml:2]: The label provides this information after triaging.

**Review by markus101 — CHANGES_REQUESTED** [review]
Don't need to prefix the title, but otherwise this looks good.

**Comment by bakerboy448** [comment]
fixed up

**Pull request merged**